test/sharness/t0230-channel-streaming-http-content-type.sh

```diff
@@ -24,6 +24,7 @@ test_ls_cmd() {
 		printf "Access-Control-Allow-Headers: X-Stream-Output, X-Chunked-Output\r\n" >>expected_output &&
 		printf "Access-Control-Expose-Headers: X-Stream-Output, X-Chunked-Output\r\n" >>expected_output &&
 		printf "Content-Type: text/plain\r\n" >>expected_output &&
+		printf "Server: go-ipfs/%s\r\n" $(ipfs version -n) >>expected_output &&
```

Comment:
Sorry for the delayed feedback -- it looks like circle-ci is failing because of the ordering of the expected headers -- now that server: is being set first, it needs to be expected first

:+1: :heart:  apart from that